### PR TITLE
Break using an alias as a mailbox loop in the email_handler.py

### PR DESCRIPTION
**Pull request by acasajus**
We don't allow to use aliases as mailbox. If that happens break the loop in the email_handler

### CODE REVIEW

File: email_handler.py [hunk 1]
```diff
@@ -693,12 +693,48 @@ def handle_forward(envelope, msg: Message, rcpt_to: str) -> List[Tuple[bool, str
...
+                    ),
+                    render(
+                        "transactional/mailbox-invalid.html",
+                        mailbox=mailbox,
+                        mailbox_url=mailbox_url,
+                        alias=alias,
+                    ),
+                    max_nb_alert=1,
+                )
+                ret.append((False, status.E525))
```
Comment (nguyenkims) [inline email_handler.py:724]: Nits: maybe we can use `continue` here so we don't need the `else` below which reduce the number of indentation 😅

File: email_handler.py [hunk 2]
```diff
@@ -1000,6 +1036,13 @@ def handle_reply(envelope, msg: Message, rcpt_to: str) -> (bool, str):
         LOG.w(f"No contact with {reply_email} as reverse alias")
         return False, status.E502
 
+    contact_alias_check = Alias.get_by(email=contact.website_email)
```
Comment (nguyenkims) [inline email_handler.py:1039]: I think we allow a contact to be an alias. For ex, a user can send an email from his alias to another user's alias.

Comment (acasajus) [reply email_handler.py]: You're right. I've removed this condition

File: email_handler.py (hunk 1 above)
Comment (acasajus) [reply email_handler.py:724]: I prefer explicit conditions over loop updates but fair enough. Added the continue and reduced the indent

**Pull request merged**